### PR TITLE
Change event.eventType.getLabel match to use event.eventType.getId

### PR DESCRIPTION
**Pull request by Johnnywoode**
- [x] Removed magic numbers from FileParser
- [x]  created scala object "EventTypeIds", where event type IDs are initialized

Reason:  To stabilize the reading of event types across Java releases/JVM vendors

### CODE REVIEW

File: cmdLine/src/main/scala/topgun/cmdline/FileParser.scala [hunk 1]
```diff
@@ -15,27 +12,25 @@ class FileParser(file: File, cmdLine: JfrParseCommandLine, totals: Totals, confi
...
-        event.getEventType.getName match {
-          case "Allocation in new TLAB" => allocation(event, true)
-          case "Allocation outside TLAB" => allocation(event, false)
-          case "Method Profiling Sample" => cpu(event)
+      val recordingFile = new RecordingFile(file.toPath)
+      //      for (event <- view.asScala) {
+      while (recordingFile.hasMoreEvents){
+        val event = recordingFile.readEvent()
+        event.getEventType.getId match {
+          case 331 => allocationInNewTlab=true; allocation(event, true) // Allocation in New TLAB
```
Comment (mkeskells) [inline cmdLine/src/main/scala/topgun/cmdline/FileParser.scala]: no magic number please - 
is this a stable number ( across Java releases /JVM vendors)
can we refer to a canonical definition of that value

**Comment by mkeskells** [comment]
looks like you base version is not in step
you need to rebase with git

**Comment by mkeskells** [comment]
this PR still needs a rebase

**Comment by Johnnywoode** [comment]
@MikeSkells 
Task already implemented in current version of mkeskells/TopGun master

**Pull request closed without merging**